src/main/java/no/unit/bibs/contents/DBClient.java

```diff
@@ -73,32 +74,31 @@ private void initDynamoDbClient(Environment environment) {
      * Adds or insert a document to dynamoDB.
      *
      * @param document the document to be inserted
-     * @throws CommunicationException when something goes wrong
      */
-    public void createContents(ContentsDocument document) throws CommunicationException {
```

Comment:
throwing a different exception might break the contract to clients. we should take this into consideration